setup/install_gatling/30_classification/40_classify_nodes_via_nc.rb

```diff
@@ -8,8 +8,7 @@ def classify_nodes(classifier)
   classifier.find_or_create_node_group_model(
       'parent' => '00000000-0000-4000-8000-000000000000',
       'name' => 'perf-agent-group',
-      'rule' => ['or', ['~', ['fact', 'clientcert'], '.*agent.*'],
-                 ['~', ['fact', 'clientcert'], "#{agent.hostname}"]],
+      'rule' => ['~', ['fact', 'clientcert'], '.*agent.*'],
```

Comment:
this uses the NC api to classify the nodes via clientcert fact as in:
https://github.com/puppetlabs/scooter/blob/master/lib/scooter/httpdispatchers/consoledispatcher.rb

Reply:
The group we want them classified into is always the same, but the class/modules we want to apply to that class change.  At some point we will certainly want multiple groups of nodes with various 'roles', but that's all future work.

Reply:
right but it class name is taken from the env at the time the rule is created...   actually I guess in this case it works because we are doing the whole run as one entity.  When we split out setup and run we will have to put this part in the run section and be sure to clear old rules.  I was still thinking of how we had two steps before.

Reply:
@RandellP Once the group is set up with the above rule, then any agent matching that regex will use that role.  We use the '.*agent.*' regex so we get any agent with the word 'agent' in the hostname, so no need to change the rules run over run.

Reply:
It was the classname I was thinking about.  when we split up the rake tasks we will want to be sure we put this code under the rake task for running the recording and I think we will also want to ensure it matches the recording.  Or maybe it has to come from the recording.  Either way, we wouldn't want it in the general setup as we would use the same setup for running different recordings with different classes.  But for now, for what we have, this is fine.